recipes/xorg-proto/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+import contextlib
+import glob
+import os
+import re
+import yaml
+
+required_conan_version = ">=1.33.0"
+
+
+class XorgProtoConan(ConanFile):
+    name = "xorg-proto"
+    description = "This package provides the headers and specification documents defining " \
+        "the core protocol and (many) extensions for the X Window System."
+    topics = ("conan", "xproto", "header", "specification")
+    license = "X11"
+    homepage = "https://gitlab.freedesktop.org/xorg/proto/xorgproto"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+
+    generators = "pkg_config"
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    @property
+    def _user_info_build(self):
+        return getattr(self, "user_info_build", self.deps_user_info)
+
+    def build_requirements(self):
+        self.build_requires("automake/1.16.3")
+        self.build_requires("xorg-macros/1.19.3")
+        self.build_requires("pkgconf/1.7.4")
+        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
+            self.build_requires("msys2/cci.latest")
+
+    def requirements(self):
+        if hasattr(self, "settings_build"):
+            self.requires("xorg-macros/1.19.3")
+
+    def package_id(self):
+        # self.info.header_only() would be fine too, but keep the os to add c3i test coverage for Windows.
```

Comment:
Please, don't do this on a regular basis (or at least, let's try to remove this afterward). We know this is an issue and we will start to test header-only packages for all the different `os+arch` soon.

Reply:
Ok.
By my knowledge, only `autoconf` has this behavior.